ota-plus-web/conf/application.conf

```diff
@@ -68,9 +68,10 @@ oidc {
     jwksUri = ${oidc.fallback.issuer}".well-known/jwks.json"
     userInfoEndpoint = ${oidc.fallback.issuer}"userinfo"
   }
+  basicAuthHeader = false
```

Comment:
basic authentication is not the only possible option, so I would prefer to have `clientAuthentication` param with two options  - `PasswordBasicAuth` and `PasswordRequestBody`, default `PasswordRequestBody`